components/calendar/src/hebrew.rs

```diff
@@ -5,14 +5,17 @@
 //! This module contains types and implementations for the Hebrew calendar.
 //!
 //! ```rust
-//! use icu::calendar::{Date, DateTime};
+//! use icu::calendar::{Ref, Date, DateTime, hebrew::Hebrew};
+//!
+//! let hebrew = Hebrew::new_always_calculating();
+//! let hebrew = Ref(&hebrew); // to avoid cloning
```

Comment:
Question: Why did you do the `Ref` thing here but not elsewhere?

Reply:
The examples here need cloning